src/frontend/components/actions/list.tsx

```diff
@@ -54,8 +55,10 @@ const List: React.FC<ActionProps> = ({ resource, setTag }) => {
     navigate({ search: search.toString() })
   }
 
+  const contentTag = getResourceElementCss(resource, 'list-content')
```

Comment:
Why not `getActionElementCss`? This is a `list` action component.

The element name could be `table-wrapper`

Reply:
done